src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
```

Comment:
nothing wrong with current robot, just inverted
```suggestion
  const invertedRobot = {};
```